### PR TITLE
Fib helper fn

### PR DESCRIPTION
**Pull request by Lotterleben**
Add a ``FIB_DEVEL_HELPER`` flag to the fib which can be enabled for debugging and testing purposes, and use it to implement ``fib_devel_get_lifetime()``. Also, add a very simple test for ``fib_devel_get_lifetime()``.

### CODE REVIEW

File: tests/unittests/tests-fib/tests-fib.c [hunk 1]
```diff
@@ -529,6 +530,32 @@ static void test_fib_14_exact_and_prefix_match(void)
...
+
+    TEST_ASSERT_EQUAL_INT(0, fib_add_entry(iface_id, (uint8_t *)addr_dst, add_buf_size - 1,
+                          addr_dst_flags, (uint8_t *)addr_nxt, add_buf_size - 1,
+                          addr_nxt_flags, 1000));
+
+    TEST_ASSERT_EQUAL_INT(0, fib_devel_get_lifetime(&lifetime, (uint8_t *)addr_dst, add_buf_size - 1));
+
+    /* assuming some ms passed during these operations... */
+    vtimer_now(&now);
+    timex_t cmp_lifetime = timex_add(now, timex_set(0, 900000));
```
Comment (BytesGalore) [inline tests/unittests/tests-fib/tests-fib.c]: how about: 
``` C
timex_t cmp_delta = timex_sub(lifetime, now);
TEST_ASSERT( cmp_delta.seconds < 1 );
```
or similar?

**Comment by BytesGalore** [comment]
I like this PR and the introduction of additional development helpers for the FIB :+1: 
And beside the lines above I tend to ACK.

_Just a thought, maybe we can extend the tests for automatic expire of entries and updating their lifetime. (but we add them afterwards)_

File: tests/unittests/tests-fib/tests-fib.c (hunk 1 above)
Comment (Lotterleben) [inline tests/unittests/tests-fib/tests-fib.c]: that would only make sure that the lifetime is bigger than the current time, right? I wanted to have the check somewhere close to the original value to make sure it gets detected when something goes completely amiss (for example wrong milisecond-microsecond translation)...

**Comment by Lotterleben** [comment]
see my lines to your lines ^^

*seems like a good idea to me*

File: tests/unittests/tests-fib/tests-fib.c (hunk 1 above)
Comment (BytesGalore) [inline tests/unittests/tests-fib/tests-fib.c]: convinced, it implicitly tests if the conversion saving the absolute time-points works as intended.
So ACK.

**Comment by BytesGalore** [comment]
lets ask travis

**Pull request merged**